h5grove/tornado_utils.py

```diff
@@ -48,18 +49,17 @@ def get(self):
             fallback=LinkResolution.ONLY_VALID,
         )
 
-        with get_content_from_file(
-            full_file_path, path, create_error, resolve_links
-        ) as content:
-            response = self.get_response(content)
```

Comment:
Implementation is a little more complicated for Tornado given the `paths` endpoint does not use the `get_content_from_file` context manager.

I had to make the `BaseHandler` a bit more generic for the `paths` endpoint and created a new `ContentHandler` that makes use of `get_content_from_file` for the other endpoints